core/src/main/resources/org/apache/spark/ui/static/timeline-view.js

```diff
@@ -50,9 +50,18 @@ function drawApplicationTimeline(groupArray, eventObjArray, startTime, offset) {
        return "#job-" + jobId;
       };
 
+      var getPathForJobEntry = function(baseElem) {
```

Comment:
No big deal but why not just inline this?

Reply:
Thanks @srowen . Yes, this is good idea, updated.